demo-app/src/types/context.ts

```diff
@@ -0,0 +1,3 @@
+export interface Context {
+  name: string
+}
```

Comment:
Technically this isn't used yet but I wanted to create placeholder for graphql context and didn't have a better idea for what to put in it so it wasn't blank and upsetting the linter.